2.4.41/run-openldap.sh

```diff
@@ -21,10 +22,10 @@ if [ ! -f /etc/openldap/CONFIGURED ]; then
         cp /usr/local/etc/openldap/DB_CONFIG /var/lib/ldap/DB_CONFIG
 
         # start the daemon in another process and make config changes
-        slapd -h "ldap:/// ldaps:/// ldapi:///" -d $OPENLDAP_DEBUG_LEVEL &
+        slapd -h "ldapi:///" -d $OPENLDAP_DEBUG_LEVEL &
```

Comment:
I think this will impact the later ldapadd command for the base org since it is not using -Y EXTERNAL. Why does it need to change?

Reply:
You're correct, I missed that last `ldapadd` command. With respect to "why the change?" I figured that it's best to not expose the LDAP server externally until you're finished configuring it. It also seems weird to be mixing `ldapi:///` and `ldap:///` access in the same script. I'll update the PR to account for that additional `ldapadd` command.